SwrveSDK/SDK/Talk/SwrveTriggerCondition.m

```diff
@@ -43,10 +43,15 @@ - (BOOL) hasFulfilledCondition:(NSDictionary *)payload {
     
     NSArray *payloadKeys = [payload allKeys];
     
-    if([payloadKeys containsObject:_key]){
-        NSString *payloadValue = [payload objectForKey:_key];
-        return ([payloadValue isEqualToString:_value]);
+    if([payloadKeys containsObject:_key]) {
         
+        if([payload objectForKey:_key] != [NSNull null]) {
```

Comment:
I've added tests to cover the cases of the user passing in "null" JSON and it goes through fine.